src/renderer/entities/governance/ui/VoteCharts/VoteChart/VoteChart.tsx

```diff
@@ -0,0 +1,75 @@
+import { useI18n } from '@app/providers';
+import { Tooltip } from '@shared/ui/Popovers';
+import { FootnoteText, HelpText } from '@shared/ui/Typography';
+import '../common/styles.css';
+
+type Props = {
+  aye: number;
+  nay: number;
+  pass: number;
+  bgColor?: string;
+  descriptionPosition?: 'tooltip' | 'bottom';
+};
+
+export const VoteChart = ({
+  aye,
+  nay,
+  pass,
+  descriptionPosition = 'tooltip',
+  bgColor = 'background-default',
+}: Props) => {
+  const { t } = useI18n();
+
+  const chartNode = (
+    <div className="vote-chart-container">
+      <div className="vote-chart-aye" style={{ width: `clamp(4px, calc(${aye}% - 2px), calc(100% - 8px))` }} />
+      <div className="vote-chart-nay" style={{ width: `clamp(4px, calc(${nay}% - 2px), calc(100% - 8px))` }} />
+      <div
+        className="vote-chart-point"
+        style={{ backgroundColor: `var(--${bgColor})`, left: `clamp(3px, ${pass}%, calc(100% - 3px))` }}
+      />
+    </div>
+  );
+
+  if (descriptionPosition === 'tooltip') {
```

Comment:
Main idea was to remove duplication of quite complex chart code. Alternative to this - create third component `VoteChartChart` (???) and reuse it in both small and large components. Also, I think if there will be a real redesign (like, pie chart instead of line), it should be new component in any case.